src/models/m_edge.erl

```diff
@@ -809,13 +816,13 @@ update_sequence(Id, Pred, ObjectIds, Context) ->
 set_sequence(Id, Pred, ObjectIds, Context) ->
     case z_acl:rsc_editable(Id, Context) of
         true ->
-            PredId = m_predicate:name_to_id_check(Pred, Context),
+            {ok, PredId} = m_predicate:name_to_id(Pred, Context),
             F = fun(Ctx) ->
                 All = z_db:q("
                             select object_id, id
                             from edge
                             where predicate_id = $1
-                              and subject_id = $2", [PredId, m_rsc:name_to_id_check(Id, Context)], Ctx),
+                              and subject_id = $2", [PredId, m_rsc:rid(Id, Context)], Ctx),
```

Comment:
I think the Erlang crash is clearer than some database constraint error, so fixed this.